.travis.yml

```diff
@@ -51,3 +51,13 @@ script:
 
 after_success:
   - if [[ "$WITH_COVERAGE" == "true" ]]; then php composer/bin/test-reporter; fi
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
@maxjacobson 

Ha, we need to set it right here, see https://travis-ci.org/codeclimate/php-test-reporter/settings:

![screen shot 2017-02-06 at 20 55 37](https://cloud.githubusercontent.com/assets/605483/22663884/be3507cc-ecae-11e6-9ef3-a5852ff7d0d0.png)

💁‍♂️ We probably need to make sure it's not displayed in the build logs.